dcicutils/env_utils.py

```diff
@@ -0,0 +1,22 @@
+FOURFRONT_STG_OR_PRD_TOKENS = ['webprod', 'blue', 'green']
+FOURFRONT_STG_OR_PRD_NAMES = ['staging', 'stagging', 'data']
+CGAP_STG_OR_PRD_TOKENS = []
+CGAP_STG_OR_PRD_NAMES = ['fourfront-cgap', 'fourfront-cgap-green', 'fourfront-cgap-blue']
+
+
+def is_cgap_env(envname):
+    return 'cgap' in envname
+
+
+def is_fourfront_env(envname):
+    return 'cgap' not in envname
+
+
+def is_stg_or_prd_env(envname):
```

Comment:
Self documenting so probably not worth adding a docstring (definitely not for the above 2 functions).

Reply:
Oh, yeah, good point. But indeed, I worked hard on the name because I started the name as just "_prd_" or "_production_" and was worried there were too many interpretations. Whatever the doc string, the name itself has to be crystal clear. I'll still put in a doc string.

Reply:
OK. I added doc strings.